.github/workflows/notify_on_broken_link_detected/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+# incomming parameters
+report_file="report.txt"
+
+# Track the broken links
+blc_track(){
+    python -m blc.blc -n -d .5 -b 5 $HOST > $report_file
```

Comment:
Can you add some comments for all the arguments you're passing here please ? 🙏🏽

Reply:
usage: blc [-h] [-c CONFIG_FILE] [-D] [-d DELAY] [-s SENDER]
                   [-p PASSWORD] [-S SMTP_SERVER] [-r RECIPIENT] [-n]
                   [-b BROWSER_SLEEP]
                   host

positional arguments:
  host                  Eg: http://example.com

options:
  -h, --help            show this help message and exit
  -c CONFIG_FILE, --config-file CONFIG_FILE
                        path to configuration file
  -D, --debug           enable the debug mode
  -d DELAY, --delay DELAY
                        It represent the delay between each request
  -s SENDER, --sender SENDER
                        It represent the email used to send the report
  -p PASSWORD, --password PASSWORD
                        It represent the password used for the login
  -S SMTP_SERVER, --smtp_server SMTP_SERVER
                        It represent the email server used to send the report
  -r RECIPIENT, --recipient RECIPIENT
                        It represent the email where send the report
  -n, --deep-scan       Enable the deep scan
  -b BROWSER_SLEEP, --browser_sleep BROWSER_SLEEP
                        Enable browser extension (if params used) and set his
                        sleep time

Reply:
haha, i know all those flags, i meant "add comment in the source code explaining what you're passing as arguments"

Reply:
ah ok, i will do it

Reply:
something like "we start the blc in deep scan with browser support enabled." is OK?

Reply:
Exactly !

Reply:
It's done !